### PR TITLE
Adding my keymap

### PR DESCRIPTION
**Pull request by Huguesmmm**
I added a Keymap for a Planck that would work on Windows and MacOS (you need to change de layer manualy).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* No issues

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Huguesmmm** [comment]
The QMK CI Build seem to have failed because of my initialisation of TAPPING_TOGGLE that is already declared in planck/ez/config.h

I need to define it because in the rev6 it is not defined. 

Should Ieave it like this or should I put a validation in my file that the variable is not define before making my initialisation?

File: keyboards/planck/keymaps/Huguesmmm/keymap_canadian_multilingual.h [hunk 1]
```diff
@@ -0,0 +1,174 @@
+// Copyright 2023 QMK
```
Comment (zvecr) [inline keyboards/planck/keymaps/Huguesmmm/keymap_canadian_multilingual.h:1]: This file should be removed as its identical to the one provided in core

File: keyboards/planck/keymaps/Huguesmmm/config.h [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#pragma once
```
Comment (waffle87) [inline keyboards/planck/keymaps/Huguesmmm/config.h:1]: This file is missing a license header.

File: keyboards/planck/keymaps/Huguesmmm/keymap.c [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+};
+
+#ifdef AUDIO_ENABLE
+  float plover_song[][2]     = SONG(PLOVER_SOUND);
+  float plover_gb_song[][2]  = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+      state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+      state = update_tri_layer_state(state, _MACLOWER, _MACRAISE, _ADJUST2);
```
Comment (waffle87) [inline keyboards/planck/keymaps/Huguesmmm/keymap.c:235]: Something tells me this is the only value ever to be returned? The core [tri layer feature](https://docs.qmk.fm/#/feature_tri_layer) should be used instead.

File: keyboards/planck/keymaps/Huguesmmm/readme.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# The Default Planck Layout
```
Comment (waffle87) [inline keyboards/planck/keymaps/Huguesmmm/readme.md:1]: This file should be removed.

**Review by waffle87 — CHANGES_REQUESTED** [review]
The directory name also needs to be lowercase.

**Pull request closed without merging**